### PR TITLE
add macos shortcut for fuzzy search and replace old jquery.hotkeys lib

### PR DESCRIPTION
**Pull request by orthagh**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes

I don't have any mac computer at disposal to test, @cconard96 maybe you have ?

### CODE REVIEW

File: src/Html.php [hunk 1]
```diff
@@ -1922,11 +1924,19 @@ private static function getPageHeaderTplVars(): array
 
         $user = Session::getLoginUserID() !== false ? User::getById(Session::getLoginUserID()) : null;
 
+        $plateform = "";
+        if (!defined('TU_USER')) {
+            $parser = new UserAgentParser();
+            $ua = $parser->parse();
+            $plateform = $ua->platform();
```
Comment (trasher) [inline src/Html.php:1931]:
```suggestion
            $platform = $ua->platform();
```

File: src/Html.php [hunk 2]
```diff
@@ -1922,11 +1924,19 @@ private static function getPageHeaderTplVars(): array
...
+        }
+
         return [
          'is_debug_active'       => $_SESSION['glpi_use_mode'] == Session::DEBUG_MODE,
          'is_impersonate_active' => Session::isImpersonateActive(),
          'founded_new_version'   => $founded_new_version,
          'user'                  => $user instanceof User ? $user : null,
+         'platform'              => $plateform,
```
Comment (trasher) [inline src/Html.php:1939]:
```suggestion
         'platform'              => $platform,
```

**Review by trasher — APPROVED** [review]

**Comment by cconard96** [comment]
I can test with a Macbook later today. I don't currently have access to one.

**Comment by cconard96** [comment]
I confirm the MacOS shortcut is working.

**Review by cconard96 — APPROVED** [review]

**Comment by orthagh** [comment]
Great thank you

**Comment by orthagh** [comment]
Do you confirm your mac is detected and the hint for the shortcut displays the good one ?

**Comment by cconard96** [comment]
Yes, the PHP-side detection works too. Displayed the correct shortcut.

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**